### PR TITLE
add tmrw-dao adapter

### PR DESCRIPTION
**Pull request by currymz**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Tomorrow DAO

##### Twitter Link:
https://x.com/tmrwdao

##### Website Link:
https://tmrwdao.com/

##### Logo (High resolution, will be shown with rounded borders):
https://raw.githubusercontent.com/TomorrowDAOProject/tomorrowDAO-interface/feat/llama/src/assets/imgs/rect-logo.svg

##### Current TVL:
10.47 M

##### Chain:
aelf

##### Short Description (to be shown on DefiLlama):
Built on aelf, TMRWDAO is a dynamic DAO platform crafted to streamline and enhance decentralised governance through innovative technology and AI integration.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Services

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/TomorrowDAOProject

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/tmrw-dao exports TVL: 
        
 
 
    aelf                      10.45 M
    
    total                    10.45 M

File: projects/tmrw-dao/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const axios = require("axios");
+
+const tvl = async (api) => {
+  const options = {
+    method: "GET",
+    url: "https://api.tmrwdao.com/api/app/networkdao/staking",
```
Comment (g1nt0ki) [inline projects/tmrw-dao/index.js:6]: I am confused, what is this staking?

Comment (currymz) [reply projects/tmrw-dao/index.js:6]: This is about aelf chain governance. Staking tokens can turn you into a BP node and gain voting rights.

Comment (g1nt0ki) [reply projects/tmrw-dao/index.js:6]: sorry, we currently are not tracking chain staking as tvl. But if your protocol has other metrics like revenue/fees, we can track it.

**Pull request closed without merging**